Bottom: d2f96b2aa5c89ba2f7904a80046d2b72f62e311a
Top:    0a1074eeca73ca92c337d4ab4d72f8f429adf0ee
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 01:50:17 +0000

Fix build warnings in gtkhid-gl.c


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index f0cf55d..accd9cd 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -396,6 +396,7 @@ ghid_draw_grid (BoxType *drawn_area)
   glEnable (GL_STENCIL_TEST);
 }
 
+#if 0
 /* XXX: Refactor this into hidgl common routines */
 static void
 load_texture_from_png (char *filename)
@@ -436,6 +437,7 @@ load_texture_from_png (char *filename)
 
   g_object_unref (pixbuf);
 }
+#endif
 
 static void
 ghid_draw_bg_image (void)
@@ -1644,8 +1646,8 @@ fill_board_outline_holes (hidGC gc, const BoxType *drawn_area)
 static void
 GhidDrawMask (int side, BoxType * screen)
 {
-  static bool first_run = true;
-  static GLuint texture;
+//  static bool first_run = true;
+//  static GLuint texture;
   int thin = TEST_FLAG(THINDRAWFLAG, PCB) || TEST_FLAG(THINDRAWPOLYFLAG, PCB);
   LayerType *Layer = LAYER_PTR (side == TOP_SIDE ? top_soldermask_layer : bottom_soldermask_layer);
   struct poly_info info;
@@ -1736,7 +1738,7 @@ GhidDrawMask (int side, BoxType * screen)
 
   gui->graphics->use_mask (HID_MASK_OFF);
 
-  first_run = false;
+//  first_run = false;
 }
 
 static void
